analyzer/src/main/kotlin/managers/SpdxDocumentFile.kt

```diff
@@ -116,7 +123,9 @@ private fun SpdxExternalDocumentReference.getSpdxPackage(workingDir: File): Spdx
                 "package. This is currently not supported yet.")
     }
 
-    val spdxPackage = spdxDocument.packages.single()
+    val spdxPackage = spdxDocument.packages.singleOrNull { it.spdxId == packageId } ?: throw IllegalArgumentException(
```

Comment:
We now have a consistency check that each package is only contained once, so we can and should use `find` instead of `singleOrNull`.

Reply:
Also formatting: Break the line before `?:`.

Reply:
Ping?

Reply:
Sorry, missed that.